### PR TITLE
Smooth opt example

### PR DESCRIPTION
**Pull request by rajibchakravorty**
Notebook with examples for RC filtered controls.

### CODE REVIEW

**Comment by saltysealion** [comment]
This PR does not look right, why does it include commits from Stefano?

**Comment by rajibchakravorty** [comment]
I merged with BLACK-474 to test the bug-fix before that branch was merged with development @saltysealion

File: examples/robust_control_optimization.ipynb [hunk 1]
```diff
@@ -46,8 +46,8 @@
     "\n",
     "# Starting a session with the API\n",
     "\n",
-    "TOKEN = 'your_qctrl_token'\n",
-    "qctrl = Qctrl(TOKEN) "
+    "TOKEN = '042ff9a9fe2137cbfa13c150de05999793b323e8'\n",
```
[the remark this answers is not in the document]
Comment (rajibchakravorty) [reply examples/robust_control_optimization.ipynb]: Yeah that needs to be removed; closing, deleting the branch and making another pr

**Comment by stevegore** [comment]
All fixed. Closing again.

**Pull request closed without merging**